src/pnpq/events.py

```diff
@@ -9,3 +9,16 @@ class Event(StrEnum):
     TX_MESSAGE_ORDERED = auto()
     TX_MESSAGE_UNORDERED = auto()
     UNCAUGHT_EXCEPTION = auto()
+
+    # General hardware errors
+    DEVICE_CONNECTED = auto()
+    DEVICE_NOT_CONNECTED_ERROR = auto()
```

Comment:
For errors, I'm of two minds. One is to remove `_ERROR` completely and rely on the log level to identify errors. The other is to move `ERROR` to the front, so that this is `ERROR_DEVICE_NOT_CONNECTED`. Why? Because it's faster to match with a regex, for one thing, or if you're reading code and looking for errors.

Thoughts?

Reply:
Good point. Given that the `UNCAUGHT_EXCEPTION` event didn't have the `ERROR`, maybe I'll remove it to keep consistency.